components/script/dom/document.rs

```diff
@@ -5441,11 +5441,9 @@ impl DocumentMethods for Document {
     }
 }
 
-fn update_with_current_time_ms(marker: &Cell<u64>) {
-    if marker.get() == 0 {
-        let time = time::get_time();
-        let current_time_ms = time.sec * 1000 + time.nsec as i64 / 1000000;
-        marker.set(current_time_ms as u64);
+fn update_with_current_time_ms(marker: &Cell<Option<CrossProcessInstant>>) {
```

Comment:
Perhaps the name needs to be changed as this no longer stores duration in milliseconds.